hazelcast/src/hazelcast/client/serialization/pimpl/SerializationConstants.cpp

```diff
@@ -18,12 +18,15 @@
 //
 
 #include "hazelcast/client/serialization/pimpl/SerializationConstants.h"
+#include "hazelcast/util/Util.h"
+#include "hazelcast/util/ILogger.h"
+#include "hazelcast/client/exception/IClassCastException.h"
 
 namespace hazelcast {
     namespace client {
         namespace serialization {
             namespace pimpl {
-                SerializationConstants *SerializationConstants::instance = NULL;
+                SerializationConstants *SerializationConstants::instance = &getInstance();
```

Comment:
why not `SerializationConstants *SerializationConstants::instance = new SerializationConstants();`?